bears/markdown/MarkdownBear.py

```diff
@@ -7,7 +7,10 @@
 @linter(executable='remark',
         use_stdin=True,
         output_format='corrected',
-        result_message='The text does not comply to the set style.')
+        result_message='The text does not comply to the set style.',
+        prerequisite_check_command=('remark-cli'),
+        prerequisite_check_fail_message='remark-cli is not installed. Please'
+        'make sure you install ``remark-cli`` and not ``remark``.')
```

Comment:
might want to add a link to the tool i.e. https://www.npmjs.com/package/remark-cli ?